### ISSUE TITLE
hide all / show all

### ISSUE THREAD

**Issue by feiss** [aframevr/aframe-inspector#451]
The new visibility icons in the scenegraph are neat, but what if you only want to show one item? Right now you have to hide every other items by clicking on the eye icon, which is quite unconvenient.

I propose adding a `shift+click` option to the eye buttons: if the item is visible, it would hide ALL items. If the item is hidden, it would show ALL items.

**Comment by fernandojsg** [comment]
Yep I was thinking about the same when I was implementing it, like a SOLO option in the sequencers, but I didn't know if it was handy using the same shortcut to recover them all
